standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/ReplChangeManager.java

```diff
@@ -467,4 +470,27 @@ static void scheduleCMClearer(Configuration conf) {
           0, MetastoreConf.getTimeVar(conf, ConfVars.REPLCMINTERVAL, TimeUnit.SECONDS), TimeUnit.SECONDS);
     }
   }
+
+  public static boolean isReplPolicySet(Database db) {
+    // Can not judge, so assuming replication is not enabled.
+    if (db == null) {
+      LOG.warn("db object passed is null. isReplPolicySet is returning false");
+      return false;
+    }
+    String replPolicyIds = getReplPolicyIdString(db);
+    return replPolicyIds != null && !replPolicyIds.isEmpty();
+  }
+
+  public static String getReplPolicyIdString(Database db) {
```

Comment:
Need not be public method as it is called within this class.

Reply:
move the repl id extract logic to database object